.github/workflows/ci.yml

```diff
@@ -21,6 +21,42 @@ jobs:
         with:
           access_token: ${{ github.token }}
 
+  build-sway-lib-core:
+    needs: cancel-previous-runs
```

Comment:
I think we should have a separate `build` job, that this and below depend on. Otherwise it takes 5 minutes to actually build `forc` so these jobs can install.

Reply:
>  This would only speed things up if we actually compiled forc as an executable binary and used that in these jobs

These two jobs use `forc` as a binary to execute `forc build` actually!

Reply:
Looks like [cross](https://github.com/cross-rs/cross) could be very useful here. 

ref: https://rust-cli.github.io/book/tutorial/packaging.html#building-binary-releases-on-ci

Reply:
Just had a chat with @adlerjohn but adding this for posterity: the `build-sway-lib-core` and `build-sway-lib-std` jobs only took a long time this time as it was the first time they were pushed and built under CI. Consecutive jobs should be closer to the existing `build-sway-examples` time (~2m) for the majority of cases, and seeing as these all run concurrently these should affect the overall time we wait for CI.